services/shuttle-poise/src/lib.rs

```diff
@@ -22,7 +21,7 @@
 //!     // Get the discord token set in `Secrets.toml`
 //!     let discord_token = secret_store
 //!         .get("DISCORD_TOKEN")
-//!         .context("'DISCORD_TOKEN' was not found")?;
```

Comment:
Why this change?

Reply:
It depends on runtime re-exporting anyhow, which is a bit unnecessary

Reply:
It just seems that the old error would have been a better practise